src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -560,6 +559,7 @@ public struct DefaultStartConfiguration : IStartConfiguration
             /// <summary>
             /// Disables ADP for the entire process for the lifetime of the process.
             /// </summary>
+            [Obsolete("This method will be removed in Dynamo 3.0 - please use Autodesk.Analytics.Core.AnalyticsUtils.DisableAnalyticsForProcessLifetime instead.")]
```

Comment:
1. so, I don't see this new method being used anywhere? When do you think it would be used?
2. If you are suggesting obsoleting and removing this method, should this be be merged to master as well?

Reply:
Forgot to change the name, thanks
I think we can obsolete the DisableADP flag in master too  When the ADP team gives us a fix, it will probably make more sense to put this functionality in a more Analytics related class (as a static flag maybe)